src/alto-ui/Dialog/__tests__/Dialog.test.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Dialog from '../Dialog';
+
+import CloseButton from '../../CloseButton';
+import Button from '../../Button';
+
+jest.mock('../../CloseButton', () => ({onClick}) => (
+  <button className="CloseButton" onClick={() => onClick()} />
+));
+
+jest.mock('../../Button', () => ({onClick}) => (
+  <button
+    className="Dialog__button"
+    onClick={() => onClick()}
+    // flat={index < buttons.length - 1}
```

Comment:
remove commented code

Reply:
Removed, I refactored the mock declarations